app/presenters/content_item_presenter.rb

```diff
@@ -103,4 +99,17 @@ def text_direction
   def native_language_name_for(locale)
     I18n.t("language_names.#{locale}", locale: locale)
   end
+
+  def merge_publishers_and_worldwide_organisations(related_navigation)
+    worldwide_orgs_and_publishers = related_navigation[:publishers]
```

Comment:
Could this be simplified to
```ruby
related_navigation[:publishers] += related_navigation.delete(:worldwide_organisations)
```
I think this would be safe because `related_navigation[:worldwide_organisations]` returns an empty array if there are no worldwide organisations. It would also remove the `:worldwide_organisations` key.

Reply:
@steventux Great, I knew there would be a simpler way of doing this 👍